inji-web/src/locales/hi.json

```diff
@@ -184,100 +184,79 @@
-      "description3": "2. इंजी वेब अनुरोध को सत्यापित करेगा और वीसी को सत्यापनकर्ता के साथ साझा करेगा, भले ही वीसी मूल रूप से किस डिवाइस से उत्पन्न या संग्रहीत किया गया हो।"
-    },
-    "item18": {
-      "title": "क्या इंजी वेब क्यूआर कोड के लिए विभिन्न प्रारूपों का समर्थन कर सकता है?",
-      "description1": "हां, इंजी वेब क्यूआर कोड उत्पन्न करने के लिए कॉन्फ़िगरेशन-आधारित दृष्टिकोण प्रदान करता है। ",
-      "description2": "1. आसान साझाकरण के लिए वीसी स्टोरेज का यूआरएल, या",
-      "description3": "2. VC डेटा स्वयं JSON प्रारूप में है (हालाँकि यह आमतौर पर सघन और कम स्कैन योग्य होता है)।"
-    },
-    "item19": {
-      "title": "साझा करने के बाद इंजी वेब क्रेडेंशियल सत्यापन कैसे संभालता है?",
-      "description1": "इंजी वेब सत्यापनकर्ता के साथ सत्यापन योग्य क्रेडेंशियल (वीसी) साझा करने के बाद:",
-      "description2": "1. सत्यापनकर्ता (उदाहरण के लिए, इंजी वेरिफाई) वीसी की प्रामाणिकता की जांच करता है।",
-      "description3": "2. यदि क्रेडेंशियल वैध है और सत्यापनकर्ता के अनुरोध से मेल खाता है, तो इसे सफलतापूर्वक सत्यापित किया जाएगा और क्रेडेंशियल स्थिति की पुष्टि की जाएगी।"
-    },
-    "item20": {
-      "title": "किस प्रकार के क्रेडेंशियल जारीकर्ता समर्थित हैं?",
-      "description1": "कोई भी क्रेडेंशियल जारीकर्ता जो OpenID4VCI प्रोटोकॉल का पालन करता है, उसे इंजी वेब में एक विश्वसनीय जारीकर्ता के रूप में सूचीबद्ध किया जा सकता है। "
-    },
-    "item21": {
-      "title": "मुझे इस रिलीज़ में पेश की गई सुविधाओं के बारे में अधिक विस्तृत जानकारी कहां मिल सकती है?",
-      "description1": "अधिक विस्तृत चरणों, सिस्टम अवलोकन और सुविधाओं पर आगे के दस्तावेज़ीकरण के लिए, कृपया संबंधित अनुभागों पर जाएँ <a href=\"https://docs.mosip.io/inji/inji-web/overview\" target=\"_blank\">इंजी वेब दस्तावेज़ीकरण</a>।"
-    },
-    "item22": {
-      "title": "मैं इंजी वेब में ऑनलाइन साझाकरण सुविधा के साथ कैसे शुरुआत करूं?",
-      "description1": "आरंभ करने के लिए, बस इंजी वेब के माध्यम से अपने सत्यापन योग्य क्रेडेंशियल तक पहुंचें।  <a href=\"https://docs.mosip.io/inji/inji-web/functional-overview/end-user-guide\" target=\"_blank\"> उपयोगकर्ता गाइड </a> और <a href=\"https://docs.mosip.io/inji/inji-web/overview\" target=\"_blank\">  प्रलेखन</a>।"
-    },
-    "item23": {
-      "title": "मैं इंजी वेब में एम्बेडेड वीसी और ऑनलाइन शेयरिंग के बीच कैसे टॉगल कर सकता हूं?",
-      "description1": "इंजी वेब में, आप क्यूआर कोड के भीतर पूर्ण सत्यापन योग्य क्रेडेंशियल (वीसी) को एम्बेड करने या मिमोटो जारीकर्ता कॉन्फ़िगरेशन में qr_code_type प्रॉपर्टी को कॉन्फ़िगर करके ऑनलाइन शेयरिंग का उपयोग करने के बीच टॉगल कर सकते हैं। ",
-      "description2": "1. EmbeddedVC: यह विकल्प संपूर्ण वीसी डेटा (JSON प्रारूप में) को सीधे क्यूआर कोड में एम्बेड करता है। ",
-      "description3": "2. OnlineSharing: यह विकल्प एक यूआरएल के साथ एक क्यूआर कोड उत्पन्न करता है जो सुरक्षित रूप से ऑनलाइन संग्रहीत वीसी को इंगित करता है। ",
-      "description4": "ये कॉन्फ़िगरेशन मिमोटो जारीकर्ता के कॉन्फ़िगरेशन में पाए जा सकते हैं। "
-    }
+    "item8":{
```

Comment:
Intendation is missing in hi.json

Reply:
missed while transaltion. 
fixed the indentation.